src/pudl/metadata/resources/eia923.py

```diff
@@ -123,6 +123,9 @@
 generation unit level up to the plant prime mover level, so as to be compatible with
 fossil fuel generation data."""
     ),
+    "core_eia923__monthly_energy_storage": (
+        """EIA-923 Monthly generation and fuel consumption for energy storage power plants. From EIA-923 Schedule 3."""
```

Comment:
I think that's a good idea. That way if there are bad fuel_units they will get flagged and we can spot fix them.

Reply:
Decided not to reference the `fuel_units` established in the `core_eia__codes_energy_sources` table because it's different for energy storage. In the context of energy storage, energy sources like pumped storage are considered to report in `mwh`. However, outside of the context of energy storage that's not the case.